src/App.tsx

```diff
@@ -1,15 +1,28 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [visibleMovies, setVisibleMovies] = useState<Movie[]>([]);
+
+  useEffect(() => {
+    setVisibleMovies(
+      moviesFromServer.filter((movie) => {
+        return (
+          movie.title.toLowerCase().includes(query.toLowerCase())
+          || movie.description.toLowerCase().includes(query.toLowerCase())
```

Comment:
You can create separate variable for ```includes(query.toLowerCase()```. Don't repeate yourselfe

Reply:
agree, but it'd be better to set `lowerCase` value in your handler right away

Reply:
I dont understand